index.html

```diff
@@ -5,6 +5,14 @@
         <link type="text/css" rel="stylesheet" href="skin.css">
     </head>
     <body>
+        <div class="nav">

```

Comment:
Looking great!

A couple of recommendations:

- There is a `nav` html element, which you could use instead of `div`. When possible, always try to use html elements that describe your content. Of course, they don't always exist, so that's when we use divs! :D Please take a look at the [html 5 element reference >](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) to see which elements are available. You can always come back to this page when you need to see if there's an element matching your content.

- As you can see, the buttons are grouped in "sections" with "titles" (`h2`, `h3`, `h4`). Since we want you and other people to be able to come to this page and know how to use your list, I recommend moving this html to its own section, with a title and usage information. Something like this:

<img width="696" alt="screen shot 2017-05-20 at 20 07 01" src="https://cloud.githubusercontent.com/assets/5296066/26278658/e9d7941e-3d97-11e7-9d7f-8e1285e78002.png">